webpack.config.js

```diff
@@ -92,7 +92,7 @@ module.exports = function (env) {
     module: {
       rules: [
         {
-          test: /\.html$/,
+          test: /\.(html|svg)$/,
           exclude: /node_modules/,
           use: {
             loader: 'file-loader',
```

Comment:
just so these files are saved in a separate folder, would you do me a favor and add this to file-loader settings:

```js
query: {
  name: 'static/[name].[hash:8].[ext]',
},
```